workflows/mark00.smk

```diff
@@ -4,8 +4,8 @@ from pyggdrasil.tree_inference import McmcConfig, TreeId, TreeType, CellSimulati
 
 ################################################################################
 # Environment variables
-DATADIR = "/cluster/work/bewi/members/gkoehn/data"
-#WORKDIR = "../data"
+#DATADIR = "/cluster/work/bewi/members/gkoehn/data"
+DATADIR = "../data"
```

Comment:
Currently - I have to change this every time I run it on a different machine - What's the best way to do this ?

Actually, have a config file here I suppose.